### PR TITLE
SDK-1034: Add unknown anchor test

### PR DESCRIPTION
**Pull request by MrBurtyyy**
- Added a test to ensure the SDK correctly handles unknown anchors as per the example from the Ruby SDK.

### CODE REVIEW

File: yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/AnchorConverterTest.java [hunk 1]
```diff
@@ -39,6 +39,7 @@
     private static final String PASSPORT_SOURCE_TYPE = "PASSPORT";
     private static final String DRIVING_LICENCE_SOURCE_TYPE = "DRIVING_LICENCE";
     private static final String YOTI_ADMIN_VERIFIER_TYPE = "YOTI_ADMIN";
+    private static final String YOTI_UNKNOWN_ISSUER = "CN=document-registration-server";
```
Comment (davidgrayston) [inline yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/AnchorConverterTest.java]: Could this be in `TestAnchors.java`?

Comment (MrBurtyyy) [reply yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/AnchorConverterTest.java]: Yep it absolutely could do - I have put it in the test file to follow convention of the other constants.  We could rename it to just UNKNOWN_ISSUER to follow the convention used.

Comment (davidgrayston) [reply yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/AnchorConverterTest.java]: Ah yes, I see what you mean - maybe it should be `DOCUMENT_REGISTRATION_ISSUER` and remain where it is - @echarrod thoughts?

Comment (echarrod) [reply yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/AnchorConverterTest.java]: I think naming it `DOCUMENT_REGISTRATION_ISSUER` and leaving it where it is seems sensible.

**Pull request merged**